### DISCUSSION TITLE
Elasticsearch for Monterey

### DISCUSSION THREAD

**Discussion by mountaindog** [Homebrew/discussions#2481] [Casks]
Not sure if this is the right spot, any timeline on getting Elasticsearch updated for Mac OS Monterey?

**Comment by carlocab** [comment]
As soon as `openjdk@11` is bottled on Monterey, probably:
```
❯ brew unbottled --tag=monterey elasticsearch
==> Populating dependency tree...
==> :monterey bottle status
elasticsearch: unbottled deps: openjdk@11
```
As for how long that will take: I don't know. You could try asking upstream.
